Modules/Backend/Resources/views/ChangePassword.blade.php

```diff
@@ -0,0 +1,49 @@
+@include('backend::header')
```

Comment:
Move all inline css into a css file

Reply:
We will include all the inline CSS into main css file after Admin part is done